app/src/androidTest/java/org/projectbuendia/client/ui/chart/PatientChartActivityTest.java

```diff
@@ -261,17 +216,33 @@ public void testEncounter_latestEncounterIsAlwaysShown() {
         for (int i = 0; i < 6; i++) {
             openEncounterForm();
 
-            String pulse = Integer.toString(i + 80);
-            String temp = Integer.toString(i + 35) + ".0";
-            String vomiting = Integer.toString(5 - i);
-            answerTextQuestion("Pulse", pulse);
+            //String pulse = Integer.toString(i + 80);
```

Comment:
Pulse is no longer in the form, right?  If the code is unused, delete it instead of commenting it out.